### PR TITLE
handled accessing thank you page directly

### LINKED ISSUE #74: ThankYou page avaible to open without sending contact form
I figured out that ThankYou action in ContactsController can easily be opened without sending the required contact form. I think that this is unnecessary action from user's perspective.
bug

### PR DESCRIPTION
**Pull request by pavelMihhailov**
Closes #74 issue.

### CODE REVIEW

File: src/PressCenters.Common/TempDataConstants.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace PressCenters.Common
+{
+    public static class TempDataConstants
+    {
+        public const string Redirected = "Redirected";
```
Comment (NikolayIT) [inline src/PressCenters.Common/TempDataConstants.cs:5]: Declare this constant in `ContactsController`

File: src/Web/PressCenters.Web/Controllers/ContactsController.cs [hunk 2]
```diff
@@ -55,11 +55,20 @@ public async Task<IActionResult> Index(ContactFormViewModel model)
...
                 model.Content);
 
+            this.TempData[TempDataConstants.Redirected] = true;
+
             return this.RedirectToAction("ThankYou");
         }
 
         public IActionResult ThankYou()
         {
+            var redirected = this.TempData[TempDataConstants.Redirected];
```
Comment (NikolayIT) [inline src/Web/PressCenters.Web/Controllers/ContactsController.cs:65]: No need to have separate variable.

File: src/PressCenters.Common/TempDataConstants.cs (hunk 1 above)
Comment (pavelMihhailov) [reply src/PressCenters.Common/TempDataConstants.cs]: Thanks for the feedback!

Comment (NikolayIT) [reply src/PressCenters.Common/TempDataConstants.cs]: And last one: may be rename the name of the temp data to `"RedirectedFromContactForm"`

Comment (pavelMihhailov) [reply src/PressCenters.Common/TempDataConstants.cs]: Done. Sorry for wasting your time, but this was my first contribution to other project 😄.

**Comment by NikolayIT** [comment]
Thanks for the pull request!

**Pull request merged**